doc/telescope.txt

```diff
@@ -213,6 +213,191 @@ actions.toggle_selection({prompt_bufnr})          *actions.toggle_selection()*
 
 
 
+================================================================================
+                                                          *telescope.previewers*
+
+Provides a Previewer table that has to be implemented by each previewer.
+To achieve this this module also provides two wrapper that abstract most of
```

Comment:
I think there should be a comma here `To achieve this, this module ...`

Reply:
Thanks :) I appreciate the review :D and all other comments. I going to update this later :)
Thank you